modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheProcessor.java

```diff
@@ -5322,6 +5334,58 @@ public CacheConfigurationEnricher enricher() {
         return enricher;
     }
 
+    /**
+     * Pages list view supplier.
+     *
+     * @param filter Filter.
+     */
+    private Iterable<CachePagesListView> pagesListViewSupplier(Map<String, Object> filter) {
+        Object cacheGrpId = filter == null ? null : filter.get(CachePagesListViewWalker.CACHE_GROUP_ID_FILTER);
```

Comment:
We shouldn't check type of the attributes on the system view level.
It's exporter's responsibility to check parameters type.